### PR TITLE
Minor doc changes

### PR DESCRIPTION
**Pull request by gspencergoog**
## Description
Just some very minor doc changes: an adjustment of my comments in https://github.com/flutter/flutter/pull/51495, and a typo fix.

### CODE REVIEW

File: packages/flutter/lib/src/foundation/diagnostics.dart [hunk 1]
```diff
@@ -3024,8 +3024,12 @@ class DiagnosticPropertiesBuilder {
 /// string and [DiagnosticsNode] debug representations describing the properties
 /// of an object.
 ///
-/// _This mixin is exists only to support plugins that require older Flutter
-/// versions: Use the identical mixin [Diagnosticable] instead for most code._
+/// _This mixin is exists only to support plugins and packages that require
```
Comment (HansMuller) [inline packages/flutter/lib/src/foundation/diagnostics.dart]: is exists => exists

File: packages/flutter/lib/src/foundation/diagnostics.dart [hunk 2]
```diff
@@ -3024,8 +3024,12 @@ class DiagnosticPropertiesBuilder {
...
+/// older Flutter versions: Use the identical mixin [Diagnosticable] instead for
+/// all new code. If you are authoring code that needs to work on the stable
```
Comment (HansMuller) [inline packages/flutter/lib/src/foundation/diagnostics.dart]: NICE

**Review by HansMuller — APPROVED** [review]
LGTM

File: packages/flutter/lib/src/foundation/diagnostics.dart (hunk 1 above)
Comment (gspencergoog) [reply packages/flutter/lib/src/foundation/diagnostics.dart]: Thanks, fixed.

**Pull request merged**